### PR TITLE
https://github.com/OpenPaaS-Suite/esn-frontend-calendar/issues/51 use esn-dav-import-client to import contact from file

### PR DESCRIPTION
**Pull request by tuanlc**
Partial resolve https://github.com/OpenPaaS-Suite/esn-frontend-calendar/issues/51

### CODE REVIEW

File: src/module-info.json [hunk 1]
```diff
@@ -52,22 +52,5 @@
...
-      },
-      "fileRoot": "frontend/app",
-      "filesGlob": [
-        "**/*.module.js",
-        "**/!(*spec).js"
-      ]
-    }
-  ]
+  "dependenceModules": []
 }
```
Comment (rezk2ll) [inline src/module-info.json:56]:
```suggestion
}

```

**Review by rezk2ll — DISMISSED** [review]

File: src/linagora.esn.contact/app/contact/contact.service.js [hunk 2]
```diff
@@ -1,177 +1,178 @@
+'use strict';
+
 const _ = require('lodash');
+const { ESNDavImportClient } = require('esn-dav-import-client/src');
```
Comment (chamerling) [inline src/linagora.esn.contact/app/contact/contact.service.js:4]: Maybe we should start doing clean stuff on our own imports. If the `esn-dav-import-client` declares its main correctly, you can remove the `/src`. For example, in the `package.json`, you must declare main like:

```
  "main": "src/index.js",
```

And then you can just do 

```js
const { ESNDavImportClient } = require('esn-dav-import-client');
```

or 

```js
import ESNDavImportClient from 'esn-dav-import-client';
```

File: src/linagora.esn.contact/app/contact/contact.service.spec.js [hunk 3]
```diff
@@ -714,11 +712,54 @@ describe('The contactService service', function() {
     });
   });
 
-  describe('The importContactsFromFile function', function() {
-    it('should call davImportService.importFromFile with right parameters to import contacts', function() {
-      var bookId = '1213';
-      var bookName = 'contacts';
-      var file = [{ type: 'text/vcard', length: 100 }];
+  describe.only('The importContactsFromFile function', function() {
```
Comment (piavgh) [inline src/linagora.esn.contact/app/contact/contact.service.spec.js:715]: .only?

File: src/linagora.esn.contact/app/contact/contact.service.js (hunk 2 above)
Comment (chamerling) [reply src/linagora.esn.contact/app/contact/contact.service.js:4]: (this is what I did for auth module)

File: src/linagora.esn.contact/app/contact/contact.service.spec.js (hunk 3 above)
Comment (chamerling) [inline src/linagora.esn.contact/app/contact/contact.service.spec.js:715]: `only` --

**Review by chamerling — CHANGES_REQUESTED** [review]

File: src/linagora.esn.contact/app/contact/contact.service.js (hunk 2 above)
Comment (tuanlc) [reply src/linagora.esn.contact/app/contact/contact.service.js]: Fixed.

BTW, I already declared the main in `esn-dav-import-client`

**Review by huy-ta — APPROVED** [review]

**Review by MichaelBailly — APPROVED** [review]

**Review by piavgh — APPROVED** [review]

**Review by chamerling — APPROVED** [review]
🤘🏻

**Pull request merged**